src/runtime/analytics-service.js

```diff
@@ -37,6 +37,14 @@ const iframeStyles = {
   display: 'none',
 };
 
+// We will wait up to 750 ms for the initial log event (iframe must load)
+const MAX_WAIT_BEFORE_LOAD = 750;
```

Comment:
@jpettitt What do you think about this max? I'd prefer 100 personally... At a certain point, it's like do we want to lose a log event or a conversion? Just my 2c

Reply:
Measuring the time it takes for logging from my computer indicates the FIRST log takes 500-600ms.  This is because an iframe has to load.  This case would only affect a user if the publisher linked to a page that immediately initiated a buy flow.

Once the iframe has loaded it takes 1-60ms to log in my measuring.  This would be the expected wait for most users (who are being redirected).

Reply:
> This case would only affect a user if the publisher linked to a page that immediately initiated a buy flow.

In that case, why not just limit the wait time to 100ms and lose the event, to increase the chances the publisher gets a subscription?